### DISCUSSION TITLE
Is Motrix development still ongoing?

### DISCUSSION THREAD

**Discussion by jtl2** [agalwood/Motrix#1152] [Q&A]
I started using Motrix last week and on my Mac and I loved it for its clean design and simplicity. However, I saw that the latest update (v1.5.x) was released almost a year ago. Is Motrix still being supported?

**Comment by therebefor** [comment]
I think it is still being supported.Motrix's developer was still active on Github at the end of 2021.I think Motrix hasn't update recently is because it didn't have big vulnerabilities or function update.Of course,this is just my guess.If you have any other question,you can ask developer by sending him an email

**Reply by jtl2** [reply]
Thanks!
